test/lib/configs/recommended.js

```diff
@@ -0,0 +1,62 @@
+const assert = require('assert');
+const { ESLint } = require('eslint');
+
+const eslintPluginAura = require('../../../lib');
+
+const eslint = new ESLint({
+  useEslintrc: false,
+  baseConfig: eslintPluginAura.configs.recommended,
+  plugins: {
+    '@salesforce/eslint-plugin-aura': eslintPluginAura,
+  },
+});
+
+describe('Recommended config', () => {
+  it('reports a parsing error when using an ES6 syntax', async () => {
```

Comment:
I'm not sure what our ES6 policy should be for this library. I know we [publicly document](https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/js_supported_es.htm?search_text=es6) that ES6 is not supported, but I am able with SFDX to define a const in an Aura controller and push to my scratch org successfully. 

We could have ES6 error by default but provide documentation for enabling ES6 locally for those who chose to use certain ES6 features, knowing the risks?

Reply:
As far as I recall, Aura parser doesn't support ES6 syntax. I think it is fair to have all ES6 syntaxes disabled.